doc/nrf/libraries/others/pcm_mix.rst

```diff
@@ -0,0 +1,32 @@
+.. _lib_pcm_mix:
+
+Pulse Code Modulation audio mixer
+#################################
+
+.. contents::
+   :local:
+   :depth: 2
+
+The Pulse Code Modulation (PCM) audio mixer lets you mix a PCM stream into another PCM stream.
+This library is useful for developing applications that offer audio features, for example using the nRF5340 Audio DK.
```

Comment:
@koffes , we need a use case/purpose for the library. This is a placeholder, please provide a better one if you think it doesn't work.

Reply:
Maybe something like:

"by for example mixing a confirmation tone together with the music that is playing"

This is what we are using pcm_mix for today in the nrf5340_audio application

Reply:
Changed